tests/app/helpers.py

```diff
@@ -493,6 +494,11 @@ def assert_no_flashes(self):
         with self.client.session_transaction() as session:
             assert not session.get("_flashes")
 
+    def assert_validation_masthead(self, res, heading="There was a problem with your answer to:"):
```

Comment:
I like this... should it go into the test-utils repo to be shared across all apps?

Reply:
Probably! 🤔

Reply:
Given the first PR on test-utils hasn't gone in yet, so there's nothing to work off, I'll add this to my list of things to do. If we can go ahead with this as-is for now, I'll refactor it out later. 👍